3rdparty/bazel/org_openssl/BUILD.bazel

```
@@ -18,11 +18,19 @@ cc_test(
         ],
         "//conditions:default": [],
     }),
+    # TODO support OpenSSL on Mac
```

```suggestion
    # TODO(malt3): support OpenSSL on Mac
```

nit: can also reference you, but I think i'm deeper in this as of today